src/txreconciliation.cpp

```diff
@@ -0,0 +1,336 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <txreconciliation.h>
+
```

Comment:
Should you introduce a namespace ? I think it would let you drop some `static` qualifiers around.